docs/setup/hooks.md

```diff
@@ -177,3 +199,35 @@ Example:
   }
 }
 ```
+
+## Experimentation
+
+It's sometimes easier to start experimenting with lakeFS webhooks, even before you have a running server to receive the calls.
+There are a couple of online tools that can intercept and display the webhook requests, one of them is Svix.
+
+1. Go to [play.svix.com](https://play.svix.com) and copy the URL address supplied by Svix.
+It should look like `https://api.relay.svix.com/api/v1/play/receive/<Random_Gen_String>/`
+   
+1. Upload the following action file to lakeFS under the path `_lakefs_actions\test.yaml` in the default branch:
+
+   ```yaml
+   name: Sending everything to Svix
+   description: Experimenting with webhooks
+   on:
+      pre-commit:
+         branches:
+      pre-merge:
+         branches:
+      post-commit:
+         branches:
+      post-merge:
+         branches:
+   hooks:
+      - id: svix
+        type: webhook
+        properties:
+           url: "https://api.relay.svix.com/api/v1/play/receive/<Random_Gen_String>/"
+   ```
+1. Commit that file to the branch.
```

Comment:
Nice to have: add a coypable lakectl command for every step.
Nice to have 2: A screenshot from Svix after the hook has run